src/main/java/co/cask/cdap/guides/workflow/EmailNotifier.java

```diff
@@ -0,0 +1,18 @@
+package co.cask.cdap.guides.workflow;
+
+import co.cask.cdap.api.workflow.AbstractWorkflowAction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This is dummy action in the Workflow used to send email notification.
```

Comment:
This is a dummy action in the Workflow that could be used to send email notification.